src/fights.ts

```diff
@@ -2404,15 +2401,13 @@ function yachtzee(): void {
     },
   ]) {
     if (available) {
-      useFamiliar(
-        Familiar.all()
-          .filter(
-            (familiar) =>
-              have(familiar) && familiar.underwater && familiar !== $familiar`Robortender`
-          )
-          .sort((a, b) => findLeprechaunMultiplier(b) - findLeprechaunMultiplier(a))[0] ??
-          $familiar.none
+      const familiarOptions = Familiar.all().filter(
+        (familiar) => have(familiar) && familiar.underwater && familiar !== $familiar`Robortender`
       );
+      const familiarChoice = familiarOptions.length
+        ? maxBy(familiarOptions, findLeprechaunMultiplier)
+        : $familiar.none;
```

Comment:
this is way more readable than it was before and i am pleased with it. my only mild concern is the use of `$familiar.none;` i think that's a defensible choice given how Guys Always Exist :tm: but it does give me a mild amount of stomach-grumblin' pain to think of a user who uses garbo where garbo literally doesn't select a familiar for them. i do not have a solution for this, to be clear. just wanted to note it.

Reply:
actually wait i do have a solution. what if we just buy and use a lep and have a note/warning for the user telling the user we're going to buy a lep?

Reply:
It would need to be an urchin urchin, but frankly this has more to do with me being worried about using `reduce` on an empty array than it does to do with default options

Again, going to declare it out of scope but an interesting thing to explore in the future.